src/utils/config-helper.ts

```diff
@@ -92,21 +91,29 @@ export async function fetchConfig (): Promise<any> {
   const siteminderLogoutUrl: string = response.data['SITEMINDER_LOGOUT_URL']
   if (siteminderLogoutUrl) {
     sessionStorage.setItem('SITEMINDER_LOGOUT_URL', siteminderLogoutUrl)
-    console.info('Set Siteminder Logout Url to: ' + siteminderLogoutUrl)
+    console.info('Set Siteminder Logout URL to: ' + siteminderLogoutUrl)
   }
 
   const addressCompleteKey: string = response.data['ADDRESS_COMPLETE_KEY'];
   (<any>window).addressCompleteKey = addressCompleteKey
-  console.info('Set Address Complete Key.')
 
   const ldClientId: string = response.data['BUSINESS_CREATE_LD_CLIENT_ID'];
   (<any>window).ldClientId = ldClientId
-  console.info('Set Launch Darkly Client ID.')
 
   const sentryEnable = response.data['SENTRY_ENABLE'];
   (<any>window).sentryEnable = sentryEnable
 
   const sentryDsn = response.data['SENTRY_DSN'];
   (<any>window).sentryDsn = sentryDsn
-  console.log('Set Sentry DSN.')
+
+  const webChatReason: string = response.data['WEBCHAT_REASON'];
+  (<any>window).webChatReason = webChatReason
+
+  const webChatStatusUrl: string = response.data['WEBCHAT_STATUS_URL'];
+  (<any>window).webChatStatusUrl = webChatStatusUrl
+  if (webChatStatusUrl) console.info('Set WebChat Status URL to: ' + webChatStatusUrl)
```

Comment:
The info message is conditional, but the `window` property is always set (otherwise it's undefined and gives an error).